docs/release-notes.md

```diff
@@ -112,6 +112,11 @@ UNRELEASED
   working properly with OpenCV 3. OpenCV 2.4 is still our primary supported
   target version of OpenCV.
 
+* Output video now runs a the full frame-rate of the input video rather than
+  slowing down during `wait_for_match`. As a side-effect the latency of the
+  video has increased by 0.5s and if the image processing is particularly slow
+  the annotations won't appear on the output video.
```

Comment:
Also: Annotations added manually when using `stbt.match` + `stbt.get_frame` intsead of `stbt.wait_for_match` will now show up in the video, as long as your processing took < 0.5s.

Reply:
P.S. The long term goal is to have these annotations rendered at playback time (e.g. when looking at a video for an old test result) instead of hard-coding them into the video at test-run time.

Reply:
I'm having difficulty wording this.  I've added a TODO item.